content/docs/rfcs/16/README.md

```diff
@@ -220,6 +221,8 @@ The `get_waku_v2_store_v1_messages` method retrieves historical messages on spec
 | ----: | :---: | :---: |----------- |
 | `pubsubTopic` | `String` | optional | The pubsub topic on which a  [`WakuMessage`](#WakuMessage) is published |
 | `contentFilters` | `Array`[[`ContentFilter`](#contentfilter)] | optional | Array of content filters to query for historical messages |
+| `startTime` | `Number`| optional | The lower bound on the [`timestamp`](/spec/14#Payloads) of queried [`WakuMessage`s](#WakuMessage). This parameter holds the Unix epoch time in seconds as a 64-bits float value.  |
```

Comment:
Might want to indicate whether these are inclusive or exclusive bounds.

Reply:
Good point! done!